examples/dataset-virtuoso/virtuoso/virtuoso.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
-
+set -e
 cd /data
-until curl --head localhost:15000 ; do echo "Waiting for Sidecar" ; sleep 3 ; done ; echo "Sidecar available";
+if [ "$ISTIO_USE" == "YES" ]; then
+    until curl --head localhost:15021 ; do echo "Waiting for Sidecar" ; sleep 3 ; done ; echo "Sidecar available" ; sleep 10 ;
```

Comment:
Modify debug messages to "waiting for Istio sidecar..." and "Istio sidecar is online."
Why sleep for 10" after verifying that Istio sidecar is up and running?